src/Common/Commands.Common.Authentication/Factories/AuthenticationFactory.cs

```diff
@@ -26,7 +26,9 @@ namespace Microsoft.Azure.Commands.Common.Authentication.Factories
 {
     public class AuthenticationFactory : IAuthenticationFactory
     {
-        public const string CommonAdTenant = "Common", DefaultMSILoginUri = "http://localhost:50342/oauth2/token";
+        public const string CommonAdTenant = "Common", 
+            DefaultMSILoginUri = "http://169.254.169.254/metadata/identity/oauth2/token", 
+            DefaultBackupMSILoginUri = "http://localhost:50342/oauth2/token";
```

Comment:
Is this for testing purposes? How would we get back a valid token from the local host?

Reply:
No, these are the real endpoints